walkthroughs/week-4-tdd/project/calendar-walkthrough.md

```diff
@@ -73,6 +75,38 @@ mvn test
 
 When all the tests pass, you can be confident that your code works!
 
+## Writing more tests
+
+To practice the principles of Test-Driven Development, it is now your turn to write some tests.
+
+You are going to be adding some new functionality to your calendar, specifically, support for optional attendees for a meeting.
+
+The basic functionality of optional attendees is that if one or more time slots exists so that both mandatory and optional attendees can attend, return those time slots. Otherwise, return the time slots that fit just the mandatory attendees.
+
+Before implementing this feature, add some new tests to tbe <walkthrough-editor-open-file
+    filePath="step/walkthroughs/week-4-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java">
```

Comment:
Have these paths changed to step (or if not, should they)?

Reply:
Discussed this Kevin and Eric and checked with the SPS team. We decided to not changes the paths (nor the packages names).